core/src/trezor/crypto/pallas.py

```diff
@@ -0,0 +1,14 @@
+"""Curve for Zcash cryptography."""
+
+from trezor import utils
+
+if utils.ZCASH_SHIELDED:
```

Comment:
in my opinion, the nicest would be to let pallas be a submodule of `trezor.curve` (since it is a curve)

but again , it would require a hierarchy
```
crypto
  __init__.py
  ...
  curve
    __init__.py
    pallas.py
```